decidim-core/app/forms/decidim/base_form.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A base form object to hold common logic, like automatically adding as
+  # attributes the params sent as context by the `FormFactory` concern.
+  class BaseForm < Rectify::Form
```

Comment:
If you include them as `attribute`, then you'll see them when you do `form.attributes`. This means that a careless consumer could rely on that to directly update the attributes of a model (`model.attributes = form.attributes`), a case scenario that is quite common when you have a 1:1 correspondence between the form and the model.

Maybe you don't have to make them `private` as they need to be accessed from the validator object, but I doubt they need to be `attributes`.